### PR TITLE
Linters setup

### PR DESCRIPTION
**Pull request by hamzaw9**
In this project we:

- Add HTML and CSS files.
- Check for linter errors.
- Ensure that the correct GitHub flow was followed for this pull request.
- Make sure work is thoroughly documented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,194 @@
...
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">Html and CSS</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] I request to consider removing the guiding statements. These are purposely to guide you what to include under that section. Therefore to make your readme professional, you should always consider removing them ☺️.

File: README.md [hunk 2]
```diff
@@ -1 +1,194 @@
-# hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
```
Comment (AlphaNtihinduka) [inline README.md]: - [ ] I have noticed that you are missing the image in your directory, kindly consider adding your image and in addition, the image used should not be the one provided by Microverse. Consider having a customized logo instead ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @hamzaw9  🙋‍♂️
Good job working on your hello Microverse project. You have done incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Your project is professional ✅
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

- _Kindly consider the inline review comments ☺️._

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Review by AbelG101 — APPROVED** [review]
# Hi @hamzaw9 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Professionally documented readme ✔
- Proper branch name ✔
- Correct use of GitHub Flow ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**